drivers/hmis/lib/ce_workflows/shared/ce_builder_utils.rb

```diff
@@ -84,7 +84,10 @@ def self.delete_template_and_associated_data(template_identifier)
       puts "Deleting existing CE data associated with #{template_identifier}"
 
       templates = Hmis::WorkflowDefinition::Template.where(identifier: template_identifier)
-      opportunities = Hmis::Ce::Opportunity.where(workflow_template_identifier: template_identifier)
+      # Find opportunities through unit groups that use this template
+      unit_groups = Hmis::UnitGroup.where(workflow_template_identifier: template_identifier).
```

Comment:
Do the `workflow_template_identifier` and `direct_referral_workflow_template_identifier` columns get cleared out? I think they should, since the intent of the method is to fully delete the template. Relates to other comment https://github.com/greenriver/hmis-warehouse/pull/5851/files#r2433246908